frontend/packages/dev-console/src/components/pipelines/pipeline-builder/PipelineBuilderForm.scss

```
@@ -5,7 +5,7 @@
 
   &__content {
     overflow: auto;
-    padding: 30px;
+    padding: 30px 30px 0px 30px;
```

```suggestion
    padding: 30px 30px 0 30px;
```

Nit: Zero is unitless.